Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -3144,7 +3144,7 @@ void MediaPlayerPrivateGStreamer::createGSTPlayBin(const URL& url)
     }
 
 #if !RELEASE_LOG_DISABLED
-    auto logIdentifier = makeString(hex(reinterpret_cast<uintptr_t>(mediaPlayerLogIdentifier())));
+    auto logIdentifier = makeString(hex(mediaPlayerLogIdentifier()));
     GST_INFO_OBJECT(m_pipeline.get(), "WebCore logs identifier for this pipeline is: %s", logIdentifier.ascii().data());
```

Comment:
Wouldn't it be easier to let `GST_INFO_OBJECT` format the identifier? As in:

```c
GST_INFO_OBJECT(m_pipeline.get(), "WebCore logs identifier for this pipeline is %" PRIx64, mediaPlayerLogIdentifier());
```

Reply:
Yes, good point. I don't know why it was done in that way.